tools/prompt-kit/src/prompts/pullRequestDescriptionGeneration.md

```diff
@@ -0,0 +1,38 @@
+# PR Description Generation Instructions
+
+These guidelines are intended to standardize how PR descriptions are generated for this codebase. They are designed to be both human- and AI-readable to ensure consistency and compatibility with tools like GitHub Copilot.
+
+First, follow these instructions to generate the **PR Title**:
+
+- The final title must be in the format: `[JIRA-ID] type(scope): concise summary of the main change`.
```

Comment:
thoughts on moving the jira id to the end? it's the least relevant for us while reviewing, so it feels a bit noisy to have it at the start